packages/contracts-bedrock/src/L2/L1BlockInterop.sol

```diff
@@ -14,7 +14,10 @@ import "src/libraries/L1BlockErrors.sol";
 enum ConfigType {
     SET_GAS_PAYING_TOKEN,
     ADD_DEPENDENCY,
-    REMOVE_DEPENDENCY
+    REMOVE_DEPENDENCY,
+    SET_BATCHER_HASH,
+    SET_GAS_CONFIG_ECOTONE,
```

Comment:
Can we replace this with two new config type events for each of the two values we hacked into the gas config with Ecotone?  E.g.

SET_BASE_FEE_SCALAR 
SET_BLOB_BASE_FEE_SCALAR

...allowing each to be updated independently.

Reply:
I am on board with this if you think that it results in less operational overhead, this just means a multicall is necessary to set both

Reply:
Alternatively just have a SET_FEE_SCALARS for both ... Important thing is to get away from the Ecotone hack that combined both fee scalars into the single "scalar" L1 attribute.

Reply:
what about keeping `SET_GAS_CONFIG_ECOTONE`, but adding `SET_BASE_FEE_SCALAR`/`SET_BLOB_BASE_FEE_SCALAR` for setting a single value?

Reply:
what about keeping `SET_GAS_CONFIG_ECOTONE`, but adding `SET_BASE_FEE_SCALAR`/`SET_BLOB_BASE_FEE_SCALAR` for setting a single value?

Reply:
Personally I don't think it's worth having two ways to do the same thing.  Probably it's best to follow Mark's advice of starting with a specification where we can argue over these API details before turning it into code?